tests/snapshots/repo__repo.snap

```diff
@@ -12,7 +12,9 @@ expression: info
       "ProjectInfo": {
         "repoName": "repo",
         "numberOfBranches": 0,
-        "numberOfTags": 1
+        "numberOfTags": 1,
+        "numberSeparator": "Dot",
+        "separator": "->"
```

Comment:
These probably don't need to be in JSON outputs (or other machine-readable outputs).

Reply:
I think serde has an attribute to tell it to ignore when converting to JSON, YAML, etc.